nuclio_sdk/platform.py

```diff
@@ -36,14 +36,17 @@ def __init__(
 
         self._control_callback = on_control_callback
 
-    async def explicit_ack(self, event):
+    async def explicit_ack(self, qualified_offset):
         """
         Notifying the processor to ack a stream message
 
-        :param event
-        :type event
+        :param qualified_offset
+        :type qualified_offset
         """
-        message = event.compile_explicit_ack_message()
+        if isinstance(qualified_offset, nuclio_sdk.Event):
```

Comment:
Maybe a bit confusing that the declared type is `qualified_offset` (should be QualifiedOffset?) but actually the function accepts `nuclio_sdk.Event` as well.